src/App.tsx

```diff
@@ -1,18 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+const ITEMS_PER_PAGE_OPTIONS = [3, 5, 10, 20];
+
+const generateItems = (count: number) => getNumbers(1, count).map((n) => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, SetPerPage] = useState(5);
+  const [currentPage, SetCurrentPage] = useState(1);
```

Comment:
```suggestion
  const [perPage, setPerPage] = useState(5);
  const [currentPage, setCurrentPage] = useState(1);
```
we usually name vars and functions in camelCase